pkg/controller/infinispan/xsite.go

```diff
@@ -178,7 +178,7 @@ func getNodePortServiceHostPort(service *corev1.Service, k *kube.Kubernetes, log
 
 	if err != nil || len(workerList.Items) == 0 {
 		// Fallback selecting everything
-		err = k.Client.List(context.TODO(), workerList, nil)
+		err = k.Client.List(context.TODO(), workerList, &client.ListOptions{})
```

Comment:
@rigazilla please pay attention, that xsite regression have found and need to be backported to 2.0.x, as I'm not sure that we need to backport whole roles assessment to the 2.0.x

Reply:
@dmvolod good spot! It's probably better to have a separate issue/bug/pr for this and a 2.0.x backport

Reply:
> good spot! It's probably better to have a separate issue/bug/pr for this and a 2.0.x backport

Yes, fixed in #588